test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/integration-test/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Self-signed cert generated by integration test
```

Comment:
Do we need this file? This is not going to be checked-in by just running the tests, right?

Reply:
Yes, this file is needed. If you run the test locally, the cert and key that are generated is stored on disk and then they're made available via a shared volume using docker-compose.